package sec04_throw;

public class ThrowExample2 {
    //JVM에게 예외를 던진다.
	public static void main(String[] args) throws Exception{
		
		throw new Exception("고의로 발생시켰음"); //예외 처리가 팔요하다.

   }
}